crates/sui-core/src/authority.rs

```diff
@@ -200,13 +202,13 @@ impl AuthorityMetrics {
             .unwrap(),
             follower_items_streamed: register_int_counter_with_registry!(
                 "follower_items_streamed",
-                "Number of transactions/signed batches streamed to followers",
+                "Number of transactions/signed .tables.batches streamed to followers",
```

Comment:
is this intended?

Reply:
oof good catch. That was find-replace issue

Reply:
Fixed!